Jenkinsfile_CNP

```diff
@@ -31,5 +31,14 @@ withPipeline(type , product, component) {
 
   enableDbMigration()
   enableSlackNotifications(channel)
-  loadVaultSecrets(secrets)
+
+  onPR {
+    loadVaultSecrets(secrets + [
+      secret('send-letter-failure-email', 'TEST_FAILURE_EMAIL')
```

Comment:
How is `TEST_FAILURE_EMAIL` used

Reply:
it's a through away value used only for PRs so [this action group](https://github.com/hmcts/send-letter-service/blob/master/infrastructure/is_down_alert_action_group.tf#L15) can be created. automatically from first run

Reply:
I can see that the checks have passed, but I don't quite understand why. I thought `secret` only creates an environment variable (in this case `TEST_FAILURE_EMAIL`) based on the Azure Key Vault secret (`send-letter-failure-email`) which must exist. The action group definition references the secret.

Reply:
based on previous comment from @timja my understanding also has been revoked. now i think it just sets the string in case actual env var is not found ¯\\\_(ツ)_/¯